crates/agentgateway/tests/common/testdata/root-cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
can you include the command used to generate this so we can recreate/modify if needed in the future